.travis.yml

```diff
@@ -112,9 +128,6 @@ script:
         /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
         /tmp/julia/bin/julia --check-bounds=yes runtests.jl libgit2-online pkg
     - cd `dirname $TRAVIS_BUILD_DIR` && mv julia2 julia &&
-        rm -rf julia/deps/build/julia-env &&
-        rm -rf julia/deps/build/libssh2-*/CMakeFiles/Makefile2 &&
-        rm -rf julia/deps/build/curl-*/config.log &&
```

Comment:
they shouldn't, since we should no longer be touching & re-running the rules that were causing this (install should be getting handled by usr-staging.tgz now and completely ignoring deps/scratch)

and confirmed by CI: https://travis-ci.org/JuliaLang/julia/jobs/157787509#L1631